src/Datadog.Trace/IScopeManager.cs

```diff
@@ -6,6 +6,8 @@ internal interface IScopeManager
 
         Scope Activate(Span span, bool finishOnClose);
 
+        void AddScopeListener(IScopeListener listener);
```

Comment:
Maybe rename to `RegisterListener` or `RegisterSubscriber`?

Reply:
I renamed it to `RegisterScopeListener`. Is that okay?